### DISCUSSION TITLE
`rpm up` show old version along new version

### DISCUSSION THREAD

**Discussion by KaeLL** [rpm-software-management/rpm#3129] [Ideas]
Make `rpm up` show the the current installed version between the Architecture and Version columns below.
![image](https://github.com/rpm-software-management/rpm/assets/4423144/4dc5f608-2422-48dd-8918-7035f2cdb285)

**Comment by ffesti** [comment]
This looks a lot like the output of `dnf up`. Are you sure you are in the right repository?

**Reply by KaeLL** [reply]
:O
I'll see myself out. Sorry.
